fedora/python-flask-whooshee.spec

```diff
@@ -5,7 +5,7 @@ Version:        0
 Release:        1%{?dist}
 Summary:        Whoosh integration
 
-License:        GPLv2+
+License:        BSD-3-Clause
```

Comment:
This would deserve some explanation in the commit description.

Reply:
Good point, documented.